project/Common.scala

```diff
@@ -0,0 +1,96 @@
+import com.typesafe.sbt.packager.Keys._
+import com.typesafe.sbt.packager.docker.DockerPlugin.autoImport.Docker
+import com.typesafe.sbt.packager.docker.{Cmd, DockerAlias}
+import sbt.Keys._
+import sbt.{Def, _}
+
+object Common {
+
+  val clocVersion = scala.io.Source.fromFile(".cloc-version").mkString.trim
+
+  private val defaultDockerInstallationPath = "/opt/codacy"
+
+  val dockerSettings: Seq[Def.Setting[_]] = Seq(
+    packageName in Docker := packageName.value,
+    dockerAlias := DockerAlias(None, Some("codacy"), name.value, Some(version.value)),
+    version in Docker := version.value,
+    maintainer in Docker := "Codacy <team@codacy.com>",
+    dockerBaseImage := "library/openjdk:8-jre-alpine",
+    dockerUpdateLatest := true,
+    defaultLinuxInstallLocation in Docker := defaultDockerInstallationPath,
+    daemonUser in Docker := "docker",
+    dockerEntrypoint := Seq(s"$defaultDockerInstallationPath/bin/${name.value}"),
+    dockerCmd := Seq(),
+    dockerCommands := dockerCommands.value.flatMap {
+      case cmd @ Cmd("ADD", _) =>
+        List(
+          Cmd("RUN", "adduser -u 2004 -D docker"),
+          cmd,
+          Cmd(
+            "RUN",
+            s"""apk update &&
+               |apk add perl &&
+               |apk add bash curl nodejs-npm &&
+               |npm install -g npm@5 &&
+               |npm install -g cloc@$clocVersion &&
```

Comment:
I think this comes from the cloc tool. to install gocyclo you have to run `go get github.com/fzipp/gocyclo`

https://github.com/fzipp/gocyclo